portfolio/src/main/webapp/index.html

```diff
@@ -3,16 +3,106 @@
+              Home
+            </a>
+            <a class="navbar-item" href="me.html">
+              About Me
+            </a>
+            <a class="navbar-item" href="photo.html">
+              Photography
+            </a>
+            <span class="navbar-item">
+              <a class="button is-inverted navbar-icon-background" href="https://linkedin.com/in/jiayi-liu-1010" target="_blank">
+                <span class="icon has-text-dark is-large">
+                  <i class="fab fa-linkedin"></i>
+                </span>
+              </a>
+            </span>
+            <span class="navbar-item">
+              <a class="button is-inverted navbar-icon-background" href="https://github.com/irisliu77" target="_blank">
+                <span class="icon has-text-dark is-large">
+                  <i class="fab fa-github-square"></i>
+                </span>
+              </a>
+            </span>
+          </div>
+        </div>
+      </div>
+    </nav>
+  </div>
+
+  <div class="hero-body"> 
+    <div class="columns is-centered">
+       <div class="column is-one-third">
+           <div class="container has-text-centered">
+              <img id="homepage_photo" src="images/homepage_photo.JPG">
+              <br>
+              <p class="self-introduction">Hello! This is Iris Liu </p>
+              <p class="self-introduction">I'm a Chapel Hill, NC based | Student | Software Engineer | Photographer | Traveler </p>
+              <p class="self-introduction">Welcome to my personal website. I hope you find this site helpful and inspiring. And feel free to contact me anytime </p>
+              <br>
+              <br>
+       </div>
     </div>
-  </body>
+  </div>
+
+  <div class="hero-foot">
+    <nav class="tabs is-boxed is-fullwidth">
+      <div class="container">
+        <ul>
+          <li style="text-align:center;">
+            <button class="button is-light project-name" id="workout-ninja-button">Workout Ninja</button>
+          </li>
+          <li style="text-align:center;">
+            <button class="button is-light project-name" id="2048-button" href="">2048</button>
+          </li>
+          <li style="text-align:center;">
+            <button class="button is-light project-name" id="tweeter-button">Mini Tweeter</button>
+          </li>
+          <li style="text-align:center;">
+            <button class="button is-light project-name" id="job-button">Job4U</button>
+          </li>
+          <li style="text-align:center;"> 
+            <button class="button is-light project-name" id="mmr-button">MMR Vaccination in US School</button>
+          </li>
+        </ul>
+      </div>
+    </nav>
+    <div class="columns is-centered">
+        <div class="column is-one-third">
+            <div class="container has-text-centered">
+                <br>
+                <img id="project-pic" src="images/Workout-Ninja.jpg">
+            </div>
+        </div>
+    </div>
```

Comment:
Currently, you haven't added the Workout-Ninja.jpg yet. Since you're adding the photos in your next PR, maybe take out this whole columns div for this PR.